etcdserver/storage.go

```diff
@@ -111,10 +110,10 @@ func readWAL(waldir string, snap walpb.Snapshot) (w *wal.WAL, id, cid types.ID,
 func upgradeDataDir(baseDataDir string, name string, ver version.DataDirVersion) error {
 	switch ver {
 	case version.DataDir0_4:
-		log.Print("etcdserver: converting v0.4 log to v2.0")
+		plog.Infof("converting v0.4 log to v2.0")
```

Comment:
hrm. it is significant internal change, but the change should not be noticed by user. so maybe info is better.